production/tanka/grafana-agent/smoke/crow/main.libsonnet

```
@@ -14,8 +14,9 @@ local volume = k.core.v1.volume;
     local this = self,
 
     _config+:: {
-      image: 'grafana/agent-crow:latest',
+      image: 'us.gcr.io/kubernetes-dev/grafana/agent-crow:dev.smoke-framework',
```

IMO it should be pointing at main now, and I think a suggestion might be able to override the PR check? 

```suggestion
      image: 'us.gcr.io/kubernetes-dev/grafana/agent-crow:main',
```